lib/mongoid/extensions/time_with_zone.rb

```diff
@@ -26,9 +26,10 @@ def mongoize
       end
 
       # This code is copied from Time class extension in bson-ruby gem. It
-      # should be removed from here when added to bson-ruby.
-      # See https://jira.mongodb.org/browse/RUBY-2846.
+      # should be removed from here when the minimum BSON version is 5+.
+      # See https://jira.mongodb.org/browse/MONGOID-5491.
       def _bson_to_i
+        return super if defined?(super)
```

Comment:
Cool, didn't realize `defined?(super)` is a thing.

Reply:
me neither!